assets/templates/cookies-preferences.tmpl

```diff
@@ -1,77 +1,70 @@
-<div class="page-intro background--gallery">
-    <div class="wrapper">
-        <div class="col-wrap">
-            <div class="col margin-top--1">
-                {{ template "partials/breadcrumb" . }}
-                <h1 class="page-intro__title margin-bottom--4 font-weight-700">
-                    {{ localise "CookiesOnONS" .Language 1}}
-                </h1>
-            </div>
-        </div>
-    </div>
-</div>
-
-<div class="wrapper adjust-font-size--18">
-    {{if .PreferencesUpdated }}
-        <div class="min-height--10 margin-top--4 cookies-banner__preferences-success">
-            <div class="font-size--18 cookies-banner__preferences-success-body">
-                <h2 class="font-weight-700 margin-top--2">{{ localise "CookiesPreferencesSaved" .Language 4 }}</h2>
-                <p class="padding-bottom--0 margin-top--1 margin-bottom--1">{{ localise "CookiesPreferencesSavedAmend" .Language 4 | safeHTML }}</p>
-            </div>
-        </div>
-    {{end}}
-
-    <p>{{ localise "CookiesP1" .Language 1}}</p>
-
-    <p>{{ localise "CookiesP2" .Language 1}}</p>
+<div class="ons-page__container ons-container">
+    <div class="ons-grid ons-u-ml-no">
+        {{ if .Page.Error.Title }}
+        {{ template "partials/error-summary" .Page.Error }}
+        {{ end }}
+        <h1 class="ons-u-fs-xxxl ons-u-mt-m ons-u-fw-b">
+            {{- localise "CookiesOnONS" .Language 1 -}}
+        </h1>
+        <div class="ons-grid__col ons-u-pl-no">
+            <div class="ons-page__main ons-u-mt-no">
+                {{if .PreferencesUpdated }}
+                    <div class="cookies-banner__preferences-success margin-bottom--2">
+                        <div class="cookies-banner__preferences-success-body">
+                            <h2>{{ localise "CookiesPreferencesSaved" .Language 4 }}</h2>
+                            <p>{{ localise "CookiesPreferencesSavedAmend" .Language 4 | safeHTML }}</p>
+                        </div>
+                    </div>
```

Comment:
Considering I struggled to understand the fundamentals of leveraging the dp-renderer to use the radios you put in, that feels like kind of a tall order for me.

Not saying I disagree with you, in fact I know your right but the idea of doing that gives me anxiety.

Reply:
It's not too dissimilar to how you would make a component in React but it can be a separate task that maybe we could pair on? 

Utilising the design system/service manual css classes and html is more important.

Reply:
Can you increase the padding between the panel and the content? It looks a bit tight
<img width="1064" alt="banner tight to content" src="https://github.com/ONSdigital/dp-frontend-cookie-controller/assets/19624419/f14f444e-e9b0-4129-ac17-b2e35509cd8e">